src/buildHTML.js

```diff
@@ -1020,13 +1024,10 @@ groupTypes.katex = function(group, options) {
         ["mord", "katex-logo"], [k, a, t, e, x], options);
 };
 
-const makeLineSpan = function(className, options) {
-    const baseOptions = options.havingBaseStyle();
-    const line = makeSpan(
-        [className].concat(baseOptions.sizingClasses(options)),
```

Comment:
Yes. The sizing classes *were* necessary because previously we wanted all rules to have the same thickness, and that thickness was specified in `katex.less` in *base-sized* `em` units. Now that we're specifying the thickness in local `em` units, no sizing classes.